src/views/base.html

```diff
@@ -20,6 +20,15 @@
   </head>
 
   <body>
+    {#
+      if the user variable is available (only the case if the page is authenticated) let them know they're logged in via a header
+    #}
+    {% if user.first_name %}
+      <header class="logged-in">
+        Logged in as {{ user.first_name }} {{ user.last_name }}
+        <a class="float-right" href="/apply/dashboard">Go to your dashboard &rarr;</a>
+      </header>
```

Comment:
This is slick!

I have a concern that this may become a nuisance on the home page, but we'll get a feel for that over time.